src/components/Avatar/Avatar.vue

```diff
@@ -66,10 +66,10 @@
 		}"
 		:style="avatarStyle"
 		class="avatardiv popovermenu-wrapper"
-		:tabindex="disableMenu ? undefined : '0'"
+		:tabindex="!hasMenu ? undefined : '0'"
 		:aria-label="avatarAriaLabel"
-		:role="disableMenu ? undefined : 'button'"
-		v-on="!disableMenu ? { click: toggleMenu } : {}"
+		:role="!hasMenu ? undefined : 'button'"
+		v-on="hasMenu ? { click: toggleMenu } : {}"
```

Comment:
```suggestion
		v-on="!hasMenu ? {} : { click: toggleMenu }"
```

For the inner-monk?

Reply:
:+1: to unify the conditions. But I think that it would be more readable to do it on `hasMenu` rather than `!hasMenu`. Or is the negated form preferred for some reason?

Reply:
then also on tabindex, I don't care about the direction, just want consistency.

Reply:
Done.